src/datachannel.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Data Channel</title>
+  <link rel="stylesheet" href="../style.css" />
+  <script src="https://sdk.virtualcareservices.net/dist/umd/vcs-realtime-sdk.min.js"></script>
+  <script src="../config.js"></script>
+</head>
+
+<body>
+  <h2>Data Channel</h2>
+  <h3>Description</h3>
+  <p>This sample app joins a room and then allow sending/receiving messages via a WebRTC Data Channel.</p>
+  <h3>Usage</h3>
+  <p>Open the <a href="./datachannel.html" target="_blank">same url</a> on another tab (or even another device/browser)
+    and then join on both tabs.</p>
+  <button id="joinLeaveBtn">Join</button>
+  <div id="videos"></div>
+
+  <div>
+    <input id="message" placeholder="Message"><br>
+    <select id="participantDropDown">
+      <option value="">All Participants</option>
+    </select><br>
+    <button id="sendMessageBtn">Send</button>
+  </div>
+
+  <script>
+    document.addEventListener('DOMContentLoaded', async () => {
+      // Import VCS_DOMAIN and ROOM_TOKEN from config.js.
+      // Edit config.js with your own vcs-domain and token.
+      await window.vcs(document);
+
+      const messageElement = document.getElementById('message');
+
+      let room;
+      let participantAddress;
+
+      const joinLeaveBtn = document.getElementById('joinLeaveBtn');
+      const videos = document.getElementById('videos');
+      const dropDownListElement = document.getElementById('participantDropDown');
+
+      sendMessageBtn.addEventListener('click', async () => {
+        data = messageElement.value;
+
+        participantAddress = dropDownListElement.value;
+
+          await room.sendMessageToParticipant(data, participantAddress);
```

Comment:
Indenting should be one less :).

Just use dropDownListElement.value here directly.
await room.sendMessageToParticipant(data, dropDownListElement.value);